lib/licenses.nix

```diff
@@ -925,6 +925,11 @@ in mkLicense lset) ({
     fullName = "No Limit Public License";
   };
 
+  notCopyrightEligible = {
+    spdxId = "Not-Copyright-Eligible";
```

Comment:
Source?  It's not in https://spdx.org/licenses/

Anyway, I think we could just use publicDomain or free here.

Reply:
> Source? It's not in https://spdx.org/licenses/

https://github.com/spdx/license-list-XML/issues/2294. While the specifics (ID, text, and whether it'll have a text at all) have not been ironed out, it looks like it'll be added in some form. I'll update the file in `lib/licenses.nix` as needed.

Reply:
> I would like to avoid adding non-standard SPDX license identifiers. We can revisit if it gets accepted.

Fair enough. Should I drop the ID and keep the license, or use `free` for now?

Reply:
Up to you.

Reply:
Switched to `free` for now.